### PR TITLE
[Doc] Expand description for facility_labels

### PR DESCRIPTION
**Pull request by karenzone**
Provide better description for `facility_labels`

### CODE REVIEW

**Comment by karenzone** [comment]
@yaauie  Thanks for providing the info. Anything else we should add? Usage example?

File: docs/index.asciidoc [hunk 1]
```diff
@@ -72,6 +72,9 @@ input plugins.
 
 Labels for facility levels. These are defined in RFC3164.
 
+You can use this option to override the integer->label mapping for syslog inputs
+that behave differently than the RFCs.
```
Comment (yaauie) [inline docs/index.asciidoc:76]: When a user overrides this, they need to provide a zero-indexed array with _all_ of their facility labels in order; when a log message arrives with a facility number with no corresponding entry, the `facility_label` is not added to the event.

The same is true for `severity_labels` option below, so we should add the note there too.

Comment (karenzone) [reply docs/index.asciidoc:76]: Thanks for the info.  I hope that I interpreted your comments correctly. 
If not, I know I can count on you to keep me honest. :-) As always, I appreciate your help.

**Comment by karenzone** [comment]
Note to self (@karenzone):  Gem publishing pending clean CI

**Comment by karenzone** [comment]
**PUBLISHED:**
Screen Shot 2020-05-19 at 5 10 16 PM

**Pull request merged**